notebook/services/kernels/kernelmanager.py

```diff
@@ -71,6 +71,16 @@ def _update_root_dir(self, proposal):
         help="""The interval (in seconds) on which to check for idle kernels exceeding the cull timeout value."""
     )
 
+    cull_connected = Bool(True, config=True,
```

Comment:
@minrk I understand taking a conservative approach but I believe that a majority of administrators that enable culling in the first place (the feature is off by default) would want connected kernels culled - provided they've exceeded the idle duration and are not busy for that period.  That said, I suppose administrators will review the four culling parameters when enabling, but still feel `cull_connected = True` is the appropriate default - IMHO.